charmhub_lp_tools/charm_project.py

```diff
@@ -219,6 +224,10 @@ def ensure_git_repository(self) -> TypeLPObject:
                         '%s does not exist, importing now from %s',
                         self.lp_project.name, self.lp_team.name,
                         self.repository)
+            if dry_run:
+                print("Git repository doesn't exist, but dry_run, bailing "
```

Comment:
the message doesn't contain enough information to understand what happened (e.g. what repository was expected to be set), I see there is a log line right above this that would have given you this information, now having said this, we should decide if we are going to be informing users via `print()` or via `logger.info()`.